tests/dialects/test_duckdb.py

```diff
@@ -356,14 +356,14 @@ def test_duckdb(self):
             "STRUCT_PACK(x := 1, y := '2')",
             write={
                 "duckdb": "{'x': 1, 'y': '2'}",
-                "spark": "STRUCT(x = 1, y = '2')",
+                "spark": "STRUCT(1 AS x, '2' AS y)",
```

Comment:
Both this and the other test below were invalid, e.g. `STRUCT(x = 1, y = '2')` is incorrect Spark in this case:

```
spark-sql (default)> select STRUCT(x = 1, y = '2');
[UNRESOLVED_COLUMN.WITHOUT_SUGGESTION] A column or function parameter with name `x` cannot be resolved. ; line 1 pos 14;
'Project [unresolvedalias(struct(col1, ('x = 1), col2, ('y = 2)), None)]
+- OneRowRelation
```